src/pages/library/Code.js

```diff
@@ -63,6 +64,22 @@ const logoTile = logoFile => div({
   }
 })
 
+export const dockstoreTile = () => div({ style: { display: 'flex' } }, [
+  logoTile(dockstoreLogo),
+  div([
+    link({ href: `${getConfig().dockstoreUrlRoot}/search?descriptorType=wdl&searchMode=files` }, 'Dockstore'),
+    div(['Browse WDL workflows in Dockstore, an open platform used by the GA4GH for sharing Docker-based tools'])
+  ])
+])
+
+export const fcMethodRepoTile = () => div({ style: { display: 'flex' } }, [
+  logoTile(firecloudLogo),
+  div([
+    link({ href: `${getConfig().firecloudUrlRoot}/?return=terra#methods` }, 'Firecloud Methods Repository'),
```

Comment:
Can these open in a new tab? I don't love that I get navigated away from Terra. I think we also usually link to a new tab if navigating outside our app, so it might be surprising that this is different.

Reply:
They were intended not to, since they round-trip you back to Terra at the end. @bradtaylor any preference?